bedrock/firefox/views.py

```diff
@@ -499,6 +499,26 @@ def sync(request):
     return l10n_utils.render(request, 'firefox/sync.html', {'version': version})
 
 
+def channel_desktop(request):
```

Comment:
I've got a change coming in pr #4514 that will remove the need for these, but I can fix them if this lands first.